engine/src/mbliphonescroller.mm

```diff
@@ -883,21 +888,32 @@ - (id)initWithInstance:(MCiOSScrollerControl*)instance
 		return nil;
 	
 	m_instance = instance;
-	
+    
 	return self;
 }
 
 - (void)scrollViewWillBeginDragging: (UIScrollView*)scrollView
 {
 	MCCustomEvent *t_event;
 	t_event = new MCiOSScrollerEvent(m_instance, MCM_scroller_begin_drag);
+    
+    CGPoint touchPoint = [scrollView.panGestureRecognizer locationInView:scrollView];
+
+    MCscreen -> device_updatemouse((int32_t)touchPoint.x,(int32_t)touchPoint.y);
+    
 	MCEventQueuePostCustom(t_event);
```

Comment:
@livecodepanos can you make sure the coordinates you get are relative to the top-left of the main view.

Reply:
Same here, it looks like they are not. I will investigate.